examples/char-rnn.cpp

```diff
@@ -191,10 +191,10 @@ static Function *createNetwork(Module &mod, PlaceholderBindings &bindings,
 
   std::vector<NodeValue> resX;
   for (unsigned i = 0; i < numSteps; i++) {
-    auto *R =
-        F->createReshape("reshapeSelector", expectedX[i], {minibatchSize, 1});
+    auto *R = F->createReshape("reshapeSelector", expectedX[i],
+                               {minibatchSize, 1}, "*");
     auto *SM = F->createSoftMax("softmax", outputNodes[i], R);
-    auto *K = F->createReshape("reshapeSM", SM, {minibatchSize, 1, 128});
+    auto *K = F->createReshape("reshapeSM", SM, {minibatchSize, 1, 128}, "*");
```

Comment:
Let's introduce something like a named constant called `ANY_LAYOUT` instead of explicitly using `"*"` everywhere.

Reply:
Also, should it be the default value for this this parameter, so that we don't need to pass it explicitly everywhere?